helpers/loanHelpers/userCollection.tsx

```diff
@@ -56,130 +72,15 @@ import {
  */
 const { format: f, formatPercent: fp, formatSol: fs } = formatNumber;
 
-/**
- * @description calculates the users total deposits
- * @params  marketReserveInfo and honeyUser
- * @returns users total deposits in market
- */
-// TODO: create types for marketReserve and honeyUser
-export async function calculateUserDeposits(
-  marketReserveInfo: any,
-  honeyUser: any
-) {
-  if (!marketReserveInfo || !honeyUser) {
-    return 0;
-  }
-
-  // await marketReserveInfo
-
-  let depositNoteExchangeRate = BnToDecimal(
-    marketReserveInfo[0].depositNoteExchangeRate,
-    15,
-    5
-  );
-
-  let depositValue = (await honeyUser.deposits().length) > 0;
-
-  if (depositValue == false) {
-    return 0;
-  } else {
-    let totalDeposits =
-      ((await honeyUser
-        .deposits()[0]
-        .amount.div(new BN(10 ** 5))
-        .toNumber()) *
-        depositNoteExchangeRate) /
-      10 ** 4;
-    return totalDeposits;
-  }
-}
-
-/**
- * @description calculates the market debt
- * @params honey reserves
- * @returns market debt
- */
-// TODO: create types for honeyReserves
-export async function calculateMarketDebt(honeyReserves: any) {
-  try {
-    const depositTokenMint = new PublicKey(
-      'So11111111111111111111111111111111111111112'
-    );
-
-    if (honeyReserves) {
-      const depositReserve = honeyReserves.filter((reserve: any) =>
-        reserve?.data?.tokenMint?.equals(depositTokenMint)
-      )[0];
-
-      const reserveState = depositReserve.data?.reserveState;
-
-      if (reserveState?.outstandingDebt) {
-        let marketDebt = reserveState?.outstandingDebt
-          .div(new BN(10 ** 15))
-          .toNumber();
-        if (marketDebt) {
-          let sum = Number(marketDebt / LAMPORTS_PER_SOL);
-          return (marketDebt = RoundHalfDown(sum));
-        }
-        return marketDebt;
-      }
-    } else {
-      return 0;
-    }
-  } catch (error) {
-    throw error;
-  }
-}
-
-export async function fetchAllowance(
-  nftPrice: number,
-  collateralNFTPositions: number,
-  honeyUser: HoneyUser,
-  marketReserveInfo: any
-) {
-  const nftCollateralValue = nftPrice * collateralNFTPositions;
-  let userLoans = 0;
-
-  if (honeyUser?.loans().length > 0) {
-    if (honeyUser?.loans().length > 0 && marketReserveInfo) {
-      userLoans =
-        (marketReserveInfo[0].loanNoteExchangeRate
-          .mul(honeyUser?.loans()[0]?.amount)
-          .div(new BN(10 ** 15))
-          .toNumber() *
-          1.002) /
-        LAMPORTS_PER_SOL;
-    }
-  }
-  return RoundHalfDown(nftCollateralValue * MAX_LTV - userLoans, 4);
-}
-
 export async function fetchLTV(totalMarketDebt: number, nftPrice: number) {
```

Comment:
still needs to be exported? @bowtiedpyro

Reply:
For now yes, will check with sdk / open request for shipping it there.